src/files/examples/fingerprint_reader_functions.py

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env python
```

Comment:
Why not Python 3?

Reply:
Cause for me its working with python... which is linked to python2.7, find better to keep on smaller for backward compatibility more than upward...